src/config.yaml

```diff
@@ -1,11 +1,13 @@
 dataset_args:
   dataset_names:
     - africa-intelligence/yahma-alpaca-cleaned-af
-    - africa-intelligence/yahma-alpaca-cleaned-zu
-    - africa-intelligence/yahma-alpaca-cleaned-xh
-    - africa-intelligence/yahma-alpaca-cleaned-tn
```

Comment:
We want to make sure that we are not losing control of the datasets that we eventually want to FT on.